golang/cosmos/x/vbank/keeper/keeper.go

```diff
@@ -39,23 +38,6 @@ func NewKeeper(
 	}
 }
 
-func (k Keeper) GetGenesis(ctx sdk.Context) types.GenesisState {
```

Comment:
Double-check that you recall why this was added to confirm that it's safe to remove, and for `SetGenesis()` too.

Reply:
Yeah, it predated `Params` and its only purpose was to persist them for later export.